webserver/templates/datasets/eval-job.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+
+{% block title %}Dataset "{{ dataset['name'] }}" by {{ author.musicbrainz_id }} - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <div id="evaluation-results">
+
+    {% from 'datasets/macros.html' import print_header %}
+    {{ print_header(dataset, author, current_user, 'evaluation') }}
+
+    {% if job['status'] == 'pending' %}
+      <div id="status" class="alert alert-info">
+        This dataset has not been evaluated yet.
+      </div>
+
+    {% elif job['status'] == 'running' %}
+      <div id="status" class="alert alert-info">
+        This dataset is being evaluated right now. Results should be available soon.
+      </div>
+
+    {% elif job['status'] == 'failed' %}
+      <div id="status" class="alert alert-danger">
+        Evaluation of this dataset has failed!
```

Comment:
Yeah, for our error monitoring we can just add calls to `logging` module and handlers (including Sentry) will record that depending on levels set there: https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/loggers.py#L7-L12.

On the website we still need a user-friendly way to display these errors. Maybe we can say "Error occurred during evaluation" and provide a way to see stack trace?